app/models/sections/transportUnit/TransportUnitsAddToListModel.scala

```diff
@@ -47,11 +51,11 @@ object TransportUnitsAddToListModel extends Enumerable.Implicits {
     )
 
     Seq(
-      radioItem(Yes, 0),
-      radioItem(NoMoreToCome, 1),
-      orDivider,
-      radioItem(MoreToCome, 2)
-    )
+      Some(radioItem(Yes, 0)),
+      if (showNoOption) Some(radioItem(NoMoreToCome, 1)) else None,
```

Comment:
I'm not sure this is included in this story however if we don't show the 'No' option then we also shouldn't show the 'or' divider. See the Documents section as an example.

Transport Units:
![image](https://github.com/hmrc/emcs-tfe-create-movement-frontend/assets/53828896/d38e0aeb-6230-4031-94f3-c218defc0640)

Documents:
![image](https://github.com/hmrc/emcs-tfe-create-movement-frontend/assets/53828896/9f80cf69-ad7c-4185-b1a8-e128778465b3)